### PR TITLE
Updated green

### PR DESCRIPTION
**Pull request by meltothewong**
Add darker greens and tones. @marcgurney Does this look good to you?

### CODE REVIEW

File: client/components/ui-design-system/base-styles/colors/index.jsx [hunk 1]
```diff
@@ -52,21 +52,21 @@ const UIColors = () => (
...
-          <p className="color-chip-hex">Hex: #c4dbb3</p>
+          <p className="color-chip-hex">Hex: #C2E0B4</p>
         </div>
       </div>
       <div className="columns small-3">
         <div className="color-chip">
           <span className="color-chip-swatch green8"></span>
           <span className="color-chip-code code">@green8</span>
-          <p className="color-chip-hex">Hex: #f0f7ec</p>
+          <p className="color-chip-hex">Hex: #F0F7EC</p>
```
Comment (Sean-Oh) [inline client/components/ui-design-system/base-styles/colors/index.jsx]: This @green8 is the old hex that you deleted - should it be #EFF6EC instead?

**Comment by meltothewong** [comment]
@Sean-Oh Hmm I don't remember making that change, but yes, that's correct. Thanks for catching that!

File: client/components/ui-design-system/base-styles/colors/index.jsx (hunk 1 above)
Comment (meltothewong) [reply client/components/ui-design-system/base-styles/colors/index.jsx]: @Sean-Oh Hmm I don't remember making that change, but yes, that's correct. Thanks for catching that!

**Comment by marcgurney** [comment]
LGTM

**Pull request merged**